build-tools/automation/yaml-templates/run-msbuild-tests.yaml

```diff
@@ -31,7 +32,8 @@ jobs:
       installLegacyDotNet: false
       restoreNUnitConsole: false
       updateMono: false
-      
+      xaSourcePath: ${{ parameters.xaSourcePath }}
```

Comment:
Do we also need to change https://github.com/xamarin/xamarin-android/blob/09daa0cf0f2f5dc366f04b4bdc61c6a758cb59d6/build-tools/automation/yaml-templates/setup-test-environment.yaml#L15 to use the checkout template?

Reply:
Correct, that will need to be done too, I did not reach that point yet, we probably want a way to use the template and pass it along like in all others